pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -81,7 +81,12 @@ func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfigura
 		return docker.AuthConfiguration{}, false
 	}
 	glog.V(3).Infof("Using %s user for Docker authentication for image %s", authConfs[0].Username, imageName)
-	return authConfs[0], true
+	return docker.AuthConfiguration{
```

Comment:
@deads2k I would rather remove usage of `go-dockerclient` instead of plumbing this. @kargakis you ok with that?